api/controllers/comment.js

```diff
@@ -0,0 +1,118 @@
+'use strict';
+
+const express = require('express');
+const router = express.Router();
+const winston = require('winston');
+const request = require('request');
+const axios = require('axios');
+
+const config = require('../config');
+const logger = winston.createLogger(config.logger.winston);
+const db = require('../models');
+const common = require('../common');
+
+
+/**
+ * @apiGroup Comment
+ * @api {get} /comment  Get Comment List of Particular Project
+ * @apiHeader {String} authorization 
+ *                              A valid JWT token "Bearer: xxxxx"
+ * @apiSuccess {Object}            List of Comments 
+ */
+
+router.get('/', common.jwt({credentialsRequired: true}), (req, res, next) =>{
+    if(!req.query.project) return next("project not defined");
```

Comment:
I think we can have different APIs for different comment type

(get) /comment/project/:projected
(get) /comment/app/:appid
etc..

And load the ID from `req.params.id`